tests/docker-compose.adjusted.yml

```diff
@@ -0,0 +1,9 @@
+services:
+
+  cvat_redis:
```

Comment:
I believe the file isn't necessary. You can define the environment variable externally. Redis should be available as redis:9379. Both cvat_redis and cvat_server are in the same network.

Reply:
We run tests on the host, not in cvat server. We should open ports for the redis container to be able to interact with it.